### DISCUSSION TITLE
Scheduled Reboot

### DISCUSSION THREAD

**Discussion by albeedelacruz** [gtxaspec/wz_mini_hacks#125] [Q&A] [answered]
Is it possible to create a script that would trigger the device to reboot on a scheduled basis?

**Comment by gtxaspec** [comment]
busybox is included, and has `crontab` and `crond`, you could use this

**Reply by albeedelacruz** [reply]
Great! Thank you

**Comment by sideup66** [comment, marked as answer]
If you need a howto, checkout the first post in this discussion. Tells you what files to make where, and how for cron to start and run a auto reboot.
https://github.com/gtxaspec/wz_mini_hacks/discussions/95
